packages/shared/src/sdk-versioning/sdk-versions/elixir.json

```diff
@@ -0,0 +1,8 @@
+{
+  "versions": [
+    {
+      "version": "0.2.0",
+      "capabilities": ["prerequisites", "semverTargeting", "bucketingV2"]
```

Comment:
The most important capability is `looseUnmarshalling`.  That means it will just ignore any new fields in the feature JSON that it doesn't recognize.  That allows us to add new functionality without breaking existing implementations.

If that is supported already, can we add it to this list?  If not, can we add it to the Elixir SDK soon?

Reply:
Yes, Elixir ignores unknown fields, will add `looseUnmarshaling`.

Reply:
Done